storage/storage_interface.ml

```diff
@@ -242,6 +242,7 @@ module Task = struct
 		subtasks: (string * state) list;
 		debug_info: (string * string) list;
 		backtrace: string;
+    cancellable: bool;
```

Comment:
Minor point: indentation looks off here.

Reply:
Good spot, the original file still indent with `<tab>`